### ISSUE TITLE
Remove engine test source code

### ISSUE THREAD

**Issue by javabrett** [yccheok/jstock#87]
Since the engine code was removed 8fad7a31784e46486c0fdaac4d387b730d18534b as discussed in #51, the test-source code under `test/org/yccheok/jstock/engine` should also be removed, to aid with building what remains.

**Comment by osascruz** [comment]
I agree, I can not compile the src files, a bit useless without the missing parts. 
Would be nice to have a compiled library so the compilation of the toplevel can works.
Thank you !
